source/_components/cast.markdown

```diff
@@ -35,6 +35,7 @@ you want to configure the IP address of the Cast device directly:
 cast:
   media_player:
   - host: 192.168.1.10
+    port: 8009
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.